src/structures/GuildChannel.js

```diff
@@ -511,11 +511,13 @@ class GuildChannel extends Channel {
    */
   get manageable() {
     if (this.client.user.id === this.guild.ownerId) return true;
-    if (VoiceBasedChannelTypes.includes(this.type)) {
-      if (!this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false)) {
-        return false;
-      }
-    } else if (!this.viewable) {
+    if (!this.viewable) {
+      return false;
+    }
+    if (
+      VoiceBasedChannelTypes.includes(this.type) &&
+      !this.permissionsFor(this.client.user).has(Permissions.FLAGS.CONNECT, false)
```

Comment:
put `this.permissionsFor(...)` into a variable so its not calculated twice unnessercerily